src/components/labels/labels.ts

```diff
@@ -0,0 +1,36 @@
+import './labels.scss';
+import WebComponent, { createElementFromString } from '../../common/WebComponent';
+
+class LabelsComponent implements WebComponent {
+  private componentElement: string = '';
+
+  render(): Element {
+    return createElementFromString(`<div class="labels">${this.componentElement}</div>`);
+  }
+
+  setup(): void {
+    return;
+  }
+
+  settingsPlayers(): string {
+    this.componentElement = 'players';
+    return this.componentElement;
+  }
+
+  settingsStyle(): string {
+    this.componentElement = '<p class="labels__settings-style">style</p>';
+    return this.componentElement;
+  }
+
+  boardWhoPlays(name: string): string {
+    this.componentElement = `${name} plays!`;
+    return this.componentElement;
+  }
+
+  boardChooseCategory(): string {
+    this.componentElement = 'choose category';
+    return this.componentElement;
```

Comment:
Just use the constructor to specify the text in label. What if I would like to have label with 20 different texts? Would I create a method for each text?